core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -667,12 +667,14 @@ void removeProperty(String tableName, String property)
    *          the name of the table
    * @return all properties visible by this table (system and per-table properties). Note that
    *         recently changed properties may not be visible immediately.
+   * @throws AccumuloSecurityException
+   *           if the user does not have permission
    * @throws TableNotFoundException
    *           if the table does not exist
    * @since 1.6.0
    */
   default Iterable<Entry<String,String>> getProperties(String tableName)
-      throws AccumuloException, TableNotFoundException {
+      throws AccumuloException, AccumuloSecurityException, TableNotFoundException {
```

Comment:
This is a breaking API change, so we probably can't do this. In other places, we sometimes wrap the checked exception in an AccumuloException, in order to avoid changing the API. It's a hack, but it gets around the restriction on breaking the API.

Reply:
Can we make this change in 3.0? That might be a good time to clean some of this stuff up with a major version release.

Reply:
Backed out API changes in a38afd6